File: JSFIles/LinearSearch.js
const linearSearch = (arr, target) =>{
    for(let i = 0; i < arr.length; i++){
        if(arr[i] === target){
            return `The number is in the position of ${i}`
        }
    }
    return "Not Found"
}


const arr = [4, 2, 5, 9, 1, 3];
const target = 1;
console.log(linearSearch(arr, target))

File: PYFiles/LinearSearch.py
def linearSearch(arr , target):
    for x in range(len(arr)):
        if arr[x] == target:
          return "The found position is {f}".format(f=x)
          
    return "None"
arr = [4, 2, 5, 9, 1, 3]
target = 1
print(linearSearch(arr, target))